packages/style-engine/class-wp-style-engine.php

```diff
@@ -30,16 +30,6 @@ class WP_Style_Engine {
 	 */
 	private static $instance = null;
 
-	/**
-	 * Instance of WP_Style_Engine_CSS_Rules_Store to hold block supports CSS rules.
-	 *
-	 * @var array<string, WP_Style_Engine_CSS_Rules_Store|null>
-	 */
-	private static $stores = array(
-		'layout-block-supports' => null,
-		'block-supports'        => null,
-	);
-
```

Comment:
We should not have a hardcoded array of stores. Instead, we can use the `WP_Style_Engine_CSS_Rules_Store::get_stores()` method to get all available/registered stores. This will make it easier to extend the class and it will work for all future iterations regardless of the store's name.

Reply:
Makes total sense. 🙇

Reply:
Just to leave a note on my thinking in relation to these hardcoded stores:

1. I was trying to find a way to control the number and the name of stores registered in Gutenberg
2. By having a hard coded list somewhere we can then control the order in which we print each store's styles onto the page. This is important for inheritance and also, later, for CSS layers.

I agree defining them here is not the right place, and we don't need to do right now anyway.

Maybe later we can think about how to tell the style engine/store class to restrict stores and the order in which we store them for the purposes of rendering to the page. 🤔 

cc @andrewserong just for a FYI

Reply:
> cc @andrewserong just for a FYI

Thanks for the heads-up, and great points! Yes, it could be a good idea for us to keep an allow-list of stores (somewhere?) to catch accidental typos / ensure that Gutenberg's styles are being added to the right place. We could even use a `_doing_it_wrong` call to flag when it's being used incorrectly?

> and we don't need to do right now anyway. Maybe later we can think about how to tell the style engine/store class to restrict stores and the order in which we store them for the purposes of rendering to the page.

Agreed.